src/bp-media/classes/class-bp-rest-media-albums-endpoint.php

```diff
@@ -905,6 +905,8 @@ public function prepare_item_for_response( $album, $request ) {
 			'title'         => $album->title,
 			'privacy'       => $album->privacy,
 			'media'         => $album->media,
+			'group_name'    => ( isset( $album->group_name ) ? $album->group_name : '' ),
```

Comment:
Do not forget to add fields **visibility** and **group_name** into the method **get_item_schema**.